.drone/drone.yml

```diff
@@ -971,8 +984,4 @@ get:
   path: infra/data/ci/loki/deploy
   name: config.json
 
----
-kind: signature
-hmac: b70be41d1a7f91c11af945a34bdbdc7a4f7613cf830c13f5438dba0bf33a1ec5
-
```

Comment:
@dannykopping `make drone` fails with `Error: you must provide the Drone server address.`